reference/configuration/kernel.rst

```diff
@@ -56,6 +56,10 @@ directory and rename it to something else (e.g. ``foo``).
 Root Directory
 ~~~~~~~~~~~~~~
 
+.. versionadded:: 3.3
+    The ``getRootDir()`` method was deprecated in Symfony 3.3 and replaced by
+    the new ``getProjectDir()`` method.
```

Comment:
What about "The `getRootDir()` method is deprecated since Symfony 3.3. Use the new `getProjectDir()` method instead."?